spec/functional/resource/group_spec.rb

```diff
@@ -133,6 +133,10 @@ def add_members_to_group(members)
       temp_resource.append(true)
       temp_resource.run_action(:modify)
       members.each do |member|
+      # TODO For some reason our temporary AIX 7.2 system does not correctly report group membership immediately after changes have been made.
```

Comment:
This can be a NOTE instead of a TODO. The remaining (and existing) comments around hopes to remove the delay are probably not going to get followed up, so we can remove those as well.